### ISSUE TITLE
Training protocols for youtube finetuning

### ISSUE THREAD

**Issue by deepcharle** [hzxie/RMNet#11]
Sorry to hear about the loss of the pretrained youtube-vos pretrained model. If you could provide the detailed training protocols for the youtube finetune so I may help you to reproduce the result.

**Comment by hzxie** [comment]
We fine-tune the model pretrained on DAVIS (not the released one) and use the subset of YouTube-VOS as the validation set.
